code/game/machinery/machinery.dm

```diff
@@ -116,16 +116,23 @@ Class Procs:
 	var/mob/living/occupant = null
 	var/unsecuring_tool = /obj/item/weapon/wrench
 	var/interact_offline = 0 // Can the machine be interacted with while de-powered.
+	var/fast_process = 0 // Process faster than other machines?

```

Comment:
yes, but I didn't want to have to copypaste that around for every machine I want running on the faster tick

instead of copypasting that block for every machine I can just flip a var to 1 and bam it does it for me

plus, if I do it this way, I'm saving myself from having to add/subtract from 2 big lists up to 3 times.